src/Sylius/Behat/Context/Setup/ShippingContext.php

```diff
@@ -142,6 +151,7 @@ private function createShippingMethod(
             $zone = $this->sharedStorage->get('zone');
         }
 
+        /** @var AbstractTranslatable | ShippingMethodInterface $shippingMethod */
```

Comment:
Should've been ``TranslatableInterface``. Main reason was to shush scrutinizer saying that this method doesn't exist in the interface. Not really needed and moreover will be pointless after merging the [TranslatableTrait](https://github.com/Sylius/Sylius/pull/4260), so reverting.